src/reports/customer-engagement.md

```diff
@@ -165,5 +165,12 @@ If you leave this field empty, a security check does not run. We do not recommen
    * 48 hours
    * 72 hours
 
-    ![System Alerts]({% link reports/assets/system.alerts.block.png %}){: .zoom}
-    _System Alerts_
\ No newline at end of file
+   ![System Alerts]({% link reports/assets/system.alerts.block.png %}){: .zoom}
+   _System Alerts_
+
+## PWA Settings
+
+Specify **PWA Storefront Base URL** field if you use a PWA storefront URL which is different from Magento's default frontend base URL. e.g. _https://pwa.yourdomain.com/_.
+
+   ![System Alerts]({% link configuration/dotdigital/assets/developer-dash-pwa-settings.png %}){: .zoom}
+   _PWA Base URL_
```

Comment:
```suggestion
## PWA settings

If you use a PWA storefront URL, specify the **PWA Storefront Base URL** (for example, `https://pwa.yourdomain.com/`). This is different from the default store frontend base URL.

![System Alerts]({% link configuration/dotdigital/assets/developer-dash-pwa-settings.png %}){: .zoom}
_PWA storefront base URL_
```

Reply:
Hello @leslietilling , thank you for the review and noticing. I have commited suggested changes and also deleted the previously added image.
Thanks!